# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fbv/linux-header.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fbv-0.99/fb_display.c.vanilla	2007-02-27 11:02:21.000000000 +0000
+++ fbv-0.99/fb_display.c	2007-02-27 11:03:26.000000000 +0000
@@ -18,8 +18,6 @@
     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 */
 
-#include <linux/fb.h>
-
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/types.h>
@@ -32,6 +30,8 @@
 #include <string.h>
 #include <errno.h>
 #include "config.h"
+#include <linux/fb.h>
+
 /* Public Use Functions:
  *
  * extern void fb_display(unsigned char *rgbbuff,
